account_bank_statement_import_online_qonto/models/online_bank_statement_provider_qonto.py

```diff
@@ -95,18 +95,18 @@ def _qonto_prepare_statement_line(
     ):
         date = datetime.strptime(transaction["settled_at"], "%Y-%m-%dT%H:%M:%S.%fZ")
         side = 1 if transaction["side"] == "credit" else -1
-        name = transaction["label"] or "/"
-        if transaction["reference"]:
-            name = "{} {}".format(name, transaction["reference"])
+        payment_ref_list = [
+            transaction["label"],
+            transaction["reference"],
+        ]
         vals_line = {
             "sequence": sequence,
             "date": date,
-            "name": name,
+            "name": " - ".join([x for x in payment_ref_list if x]) or "/",
```

Comment:
Having done some fancy statement matching in my time it seems to me that having the dash in between can lead to some noise when splitting up the line name by whitespace so I have a small preference for joining by space only.

Reply:
I always prefer to have a more distinct joining char, being `-`, `;` or whatever. If not, you don't differentiate if the content comes from one place or another. Do you have an example of why it's better the space?

Reply:
It may be a theoretical case but if you have statement references like "1 - 2" and you have outstanding items with references "3 - 4" and you split your references by whitespace you could have a match on the `dash.`
But like I said, it's not blocking for me.

Reply:
OK, let's just keep it.